packages/currency/src/index.ts

```diff
@@ -150,6 +125,7 @@ export function getAllSupportedCurrencies(): {
  * @param currency
  *
  * @returns the hash of the currency
+ * @deprecated use new Currency().getHash() instead
```

Comment:
should this method call `new Currency(currency).getHash()` to avoid having 2 implementations and a risk of divergence?

Reply:
You're right, I forgot. Thanks for spotting !